nifi-api/src/main/java/org/apache/nifi/reporting/Bulletin.java

```diff
@@ -98,6 +99,14 @@ public void setGroupName(String groupName) {
         this.groupName = groupName;
     }
 
+    public String getGroupPath() {
```

Comment:
Can we add this to the BULLETINS table in the sql-reporting bundle too? Would be helpful for queries.